conf/proxysql.conf.example

```diff
@@ -150,3 +150,17 @@ mysql_users =
 (
 [% mysql_users %]
 )
+
+scheduler =
+(
+  {
+     id=1
+     active=1
+     interval_ms=5000
+     filename="/proxysql-read-only-handler.sh"
+     arg1="{\"username\":\"admin\", \"password\":\"admin\", \"host\":\"127.0.0.1\", \"port\":6032}"
+     arg2="{\"username\":\"[% mysql_pf_user %]\", \"password\":\"[% mysql_pf_pass %]\"}"
+     arg3="{\"read_write_hostgroup\":10, \"read_hostgroup\":30, \"rule_ids\":\"100,400\"}"
+     arg4="/proxysql-read-only-handler.log"
+  }
+)
```

Comment:
change the file permissions to make it executable.

Reply:
It is already via the Dockerfile:
https://github.com/inverse-inc/packetfence/pull/7153/files#diff-66d4b4df5930c5daf1764ead4c20e20ab7406ea37b521051cf1b860cdaa5de93R79